web/src/frontend/src/digisos/skjema/arbeidUtdanning/Studie.tsx

```diff
@@ -11,8 +11,8 @@ const Studie: React.StatelessComponent = () => {
 	return (
 		<JaNeiSporsmalFaktum faktumKey={studie.faktum}>
 			<SporsmalFaktum faktumKey={studerer.faktum}>
-				<RadioFaktum faktumKey={studerer.faktum} value="heltid" />
-				<RadioFaktum faktumKey={studerer.faktum} value="deltid" />
+				<RadioFaktum id="studerer_radio_heltid" faktumKey={studerer.faktum} value="heltid" />
```

Comment:
Kunne en brukt faktum-key på alle? Da kunne du kanskje latt FaktumComponent legge det på automatisk, slik at du slipper å gjøre det på alle elementene

Reply:
Og får ikke allerede RadioFaktum et name som kan brukes ganske unikt for targeting?

Reply:
Jeg la på egne id'er for å få kortere og mer selvforklarende navn på radioknapp'ene. Hvis det ikke ligger id på en RadioFaktum, så brukes faktum-id som id. Som en slags failoverløsning.